src/input/wiimote_manager.cpp

```diff
@@ -385,12 +385,12 @@ int WiimoteManager::askUserToConnectWiimotes()
 {
     new MessageDialog(
 #ifdef WIN32
-        _("Connect your wiimote to the Bluetooth manager, then click on Ok. "
+        _("Connect your Wiimote to the Bluetooth manager, then click on OK. "
                   "Detailed instructions at supertuxkart.net/Wiimote"),
 #else
-        _("Press the buttons 1+2 simultaneously on your wiimote to put "
-          "it in discovery mode, then click on Ok. "
-                  "Detailed instructions at supertuxkart.net/Wiimote"),
+        _("Press the 1 and 2 buttons on your Wiimote simultaneously to put "
+          "it in discovery mode, then click on OK. "
+                  "Detailed instructions at http://supertuxkart.net/Wiimote"),
```

Comment:
I think the point here was to remove the prefix to make it the same as the line 389.